### PR TITLE
Multiple minor fixes

### PR DESCRIPTION
**Pull request by viliam-durina**

### CODE REVIEW

File: src/test/java/com/hazelcast/jdbc/ParameterListTest.java [hunk 1]
```diff
@@ -0,0 +1,4 @@
+import static org.junit.jupiter.api.Assertions.*;
+class ParameterListTest {
```
Comment (SeriyBg) [inline src/test/java/com/hazelcast/jdbc/ParameterListTest.java:2]: Is this class used anywhere?

File: src/main/java/com/hazelcast/jdbc/Driver.java [hunk 2]
```diff
@@ -39,11 +38,14 @@
     @Override
     public Connection connect(String url, Properties info) throws SQLException {
         if (url == null) {
-            throw new SQLException("Url is null");
+            throw new SQLException("URL is null");
         }
-        JdbcUrl jdbcUrl = JdbcUrl.valueOf(url, info);
-        if (jdbcUrl == null) {
-            return null;
+        JdbcUrl jdbcUrl;
```
Comment (SeriyBg) [inline src/main/java/com/hazelcast/jdbc/Driver.java:43]: This seems to violate the specification, as according to the Javadoc: _The driver should return "null" if it realizes it is the wrong kind of driver to connect to the given URL._
But here the `SQLException` will be thrown

**Review by SeriyBg — CHANGES_REQUESTED** [review]

File: src/test/java/com/hazelcast/jdbc/ParameterListTest.java (hunk 1 above)
Comment (viliam-durina) [reply src/test/java/com/hazelcast/jdbc/ParameterListTest.java:2]: Looks like a mistake when committing, will fix.

File: src/main/java/com/hazelcast/jdbc/Driver.java (hunk 2 above)
Comment (viliam-durina) [reply src/main/java/com/hazelcast/jdbc/Driver.java:43]: Yeah, i didn't read the contract fully, I assumed that the manager calls `acceptsUrl` first before this method. I'll add the check.

**Review by SeriyBg — APPROVED** [review]

**Pull request merged**